packages/CB-serverless-backend/utils/getResponse.js

```diff
@@ -0,0 +1,9 @@
+const getResponse = (data) => {
```

Comment:
can be renamed to `getSuccessResponse` and contents of `renderServerError.js`  can be added here as `getErrorResponse` since all are responses basically.